### ISSUE TITLE
Pull to refresh List View?

### ISSUE THREAD

**Issue by StephenKDS** [ianharrigan/haxeui#264]
Any chance of getting a Pull To Refresh list view control?

looks like a list view, but when at the top of it scrolling, pulls down and let go. Refreshes the content in the list box. For example, look at the twitter, facebook, snap chat type apps.

**Comment by ianharrigan** [comment]
Yeah, this is a nice idea. Im going to be giving the list view a few tweaks soon(ish) so will try and think about this then.

Cheers,
Ian

**Comment by ianharrigan** [comment]
Should be available now, example here: http://haxeui.org/try.jsp?layoutId=6zpn9AK

**Comment by StephenKDS** [comment]
WIcked dude! just in time as I have started building my mobile app using HaxeUI :)

**Comment by dhemitus** [comment]
is any example pull to refresh from the bottom of the list?

**Comment by ianharrigan** [comment]
Im not quite sure how that would work, haxeui doesnt support that, but logically it doesnt really make sense to me either, i mean, if you had 50 items on the list and you "pull" when you are at the bottom, wouldnt you expect the list to move up?

**Comment by dhemitus** [comment]
yes, i mean increment data from the bottom not from the top of list
